### PR TITLE
Update contact-pstn-service-desk.md

### PR DESCRIPTION
**Pull request by ErsancMS**
Proposed change for this page so users can review our scope better.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 1]
```diff
@@ -27,26 +27,30 @@ description: "When you are getting phone numbers or porting (transferring) numbe
 
 # PSTN Service Desk Contact
 
-If you need assistance transferring/porting your existing phone numbers or acquiring new phone numbers for Microsoft Teams, please send us an email and include as many details as possible:
-  - Your [Organization ID (Tenant ID) or currently active Microsoft domains.]((https://docs.microsoft.com/onedrive/find-your-office-365-tenant-id)
+The PSTN Service Desk team currently has 3 email addresses for support, which are presented in the following table:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:30]:
```suggestion
The PSTN Service Desk team currently has three email addresses for support, which are presented in the following table:
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 2]
```diff
@@ -27,26 +27,30 @@ description: "When you are getting phone numbers or porting (transferring) numbe
...
+| **I want to transfer/port existing numbers..**  | **I want to order new numbers for..** |
+|:-----:|:------:|
+| [**Inside the United States or Canada** email us at ptn@microsoft.com](mailto:ptn@microsoft.com)| [**Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, email us at ptneu@microsoft.com](mailto:ptneu@microsoft.com)|
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:33]:
```suggestion
| [**Inside the United States or Canada**, email us at ptn@microsoft.com](mailto:ptn@microsoft.com)| [**Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, email us at ptneu@microsoft.com](mailto:ptneu@microsoft.com)|
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 3]
```diff
@@ -27,26 +27,30 @@ description: "When you are getting phone numbers or porting (transferring) numbe
...
+|[**Outside the United States or Canada**: email us at ptneu@microsoft.com](mailto:ptneu@microsoft.com)| [**Rest of the World**, email us at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com)|
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:34]:
```suggestion
|[**Outside the United States or Canada**, email us at ptneu@microsoft.com](mailto:ptneu@microsoft.com)| [**Rest of the World**, email us at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com)|
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 4]
```diff
@@ -27,26 +27,30 @@ description: "When you are getting phone numbers or porting (transferring) numbe
...
+
+If you need assistance **transferring/porting your existing phone numbers to Microsoft Teams**, please include as many details as possible:
+  - Your [Organization ID (Tenant ID) or currently active Microsoft domains.](https://docs.microsoft.com/onedrive/find-your-office-365-tenant-id)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:37]:
```suggestion
  - Your [Organization ID (Tenant ID) or currently active Microsoft domains](https://docs.microsoft.com/onedrive/find-your-office-365-tenant-id).
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**